src/sp/pl_session.cpp

```diff
@@ -74,9 +76,7 @@ namespace cubpl
     , m_all_session_params_required {true}
     , m_last_conn_epoch (-1)
     , m_stack_idx {-1}
-    , m_is_interrupted (false)
-    , m_interrupt_id (NO_ERROR)
-    , m_is_running (false)
+    , m_interrupt_id (0)
```

Comment:
기존 방식이 더 명확한 것 같습니다.
```suggestion
    , m_interrupt_id (NO_ERROR)
```

Reply:
네. 원복 하겠습니다. 
if (m_interrupt_id)  이렇게 간단하게 쓰려고 바꾼 것입니다만, 
(이렇게 쓰면 NO_ERROR 값이 나중에라도 혹시 0이 아니게 바뀐다면 문제가 됩니다. ) 
if (m_interrupt_id != NO_ERROR) 가 더 명확하네요.